lib/omnicache/store.rb

```diff
@@ -161,6 +189,20 @@ def size
 
     private
 
+    def with_tracing(resource, &block)
+      if defined?(Datadog::Tracing)
+        Datadog::Tracing.trace(
+          "omnicache",
```

Comment:
[DD documentation](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/ruby/#manual-instrumentation) has this to say about this `name` arg:
>  `name` should be a `String` that describes the generic kind of operation being done (e.g. `'web.request'`, or `'request.parse'`)

Should this value change to `"omnicache.#{resource}"`?

Reply:
Consistency is prly best